setuptools/tests/test_bdist_egg.py

```diff
@@ -64,3 +65,18 @@ def test_exclude_source_files(self, setup_context, user_override):
         names = list(zi.filename for zi in zip.filelist)
         assert 'hi.pyc' in names
         assert 'hi.py' not in names
+
+    def test_eggsecutable_warning(self, setup_context, user_override):
+        dist = Distribution(dict(
+            script_name='setup.py',
+            script_args=['bdist_egg'],
+            name='foo',
+            py_modules=['hi'],
+            entry_points={
+                'setuptools.installation':
+                    ['eggsecutable = my_package.some_module:main_func']},
+        ))
+        with contexts.quiet():
```

Comment:
Is this necessary? It seems like it circumvents `pytest`'s stdin/stdout capture. I'm wondering if it was a holdover from the `unittest` days, maybe?

Reply:
Well, removing that line still makes the test pass, so I can remove it. However, *all* other tests in this module follow the same pattern and have that line.